INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
+
+## Prerequisites
```

Comment:
>Can you verify that these contain the required header files and your's are not coming from somewhere else? :)

Timing Problem :) - I don't have any Qt headers installed in /usr/include. For the above reason, header files always come from the `$QTDIR`. Our `mgv-install-script` asks for that on startup :).